executors/node/collator_nonignorable.js

```diff
@@ -0,0 +1,44 @@
+// The Collator for nonignorable short tests.
```

Comment:
Question: can you explain why you made this a new executor instead of adding it to the existing coll_shift_short executor? It would be nice to make clearer when to add things to an existing executor versus creating a new executor

Reply:
Yup, I've refactored this to use the same collator.js functions with a parameter. Same for ICU4X version.

Reply:
I think you should keep a single collator testType.